pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -313,6 +318,7 @@ func (p *PriorityQueue) Add(pod *v1.Pod) error {
 	if err := p.podBackoffQ.Delete(pInfo); err == nil {
 		klog.ErrorS(nil, "Error: pod is already in the podBackoff queue", "pod", klog.KObj(pod))
 	}
+	klog.V(5).InfoS("Pod moved", "pod", klog.KObj(pod), "event", PodAdd, "from", "", "to", activeQName)
```

Comment:
nit: maybe specify it's "from" `New` (or `Incoming`)

Reply:
Changed it `New`